src/configs/config.ts

```diff
@@ -0,0 +1,27 @@
+import dotenv from 'dotenv';
```

Comment:
what was the reason you wanted to have `dotenv` in production w/ lambdas?

Reply:
This particular thing I have inherited from my .net core experience. In .net core, developers prefer to have appsettings file and one can even override the env variable by using appsettings json file. Its just more continent to use settings file because .net framework itself supports better maintainability of by allowing them to be broken down on cohesion basis. But seeing your comment on this thing making me realize things work differently in nodejs. In simple words, it's just due to past experience.